### DISCUSSION TITLE
[4.0] API for Media

### DISCUSSION THREAD

**Discussion by Razzo1987** [joomla/joomla-cms#27244] [2. Cold Features]
### Describe the solution you'd like

A set of API for:

- upload imege (or media) in "imeges" directory
- upload imege (or media) in sub-directory
- create a new folder/subfolder in "images" directory
- list the folder tree of the "imeges" directory
- list the media in directory/subdirectory

### Additional context

With these API it will be possible to insert images in Joomla that can then be used in the creation of the articles

**Comment by chetanmadaan** [comment]
I'll be happy to test, when and if this is done.
